### PR TITLE
Gallery component 

### PR DESCRIPTION
**Pull request by 1makedadavis**

### CODE REVIEW

File: client/src/Components/Overview/Gallery.jsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+
+
+const Gallery = () => {
+  const [productURL, setProductUrl] = useState('');
+  const [styles, setStyles] = useState([]);
+  const [currentStyle, setCurrentStyle] = useState([]);
+
+  const getProduct = () => {
+    axios.get('/products/40348/styles')
```
Comment (apriljgranzow) [inline client/src/Components/Overview/Gallery.jsx:12]: This part can be refactored to take a variable product id

File: client/src/Components/Overview/Gallery.jsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      .then((res) => {
+        setProductUrl(res.data.results[0].photos[0].url);
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  };
+
+  useEffect(() => {
+    axios.get('/products/40348/styles')
```
Comment (apriljgranzow) [inline client/src/Components/Overview/Gallery.jsx:22]: consider moving this into its own function

**Review by apriljgranzow — CHANGES_REQUESTED** [review]
Please move any styles out of the index.html and only put them in the CSS file.

File: client/src/Components/Overview/Gallery.jsx (hunk 1 above)
Comment (1makedadavis) [reply client/src/Components/Overview/Gallery.jsx:10]: I am working on that.

File: client/src/Components/Overview/Gallery.jsx (hunk 2 above)
Comment (1makedadavis) [reply client/src/Components/Overview/Gallery.jsx:20]: Can we talk about this?

**Comment by 1makedadavis** [comment]
Hi April, I made changes can you review please. Thanks.

**Review by apriljgranzow — APPROVED** [review]

**Pull request merged**